### ISSUE TITLE
Accessibility: Replace H1's in non-page patterns

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyfive#186]
**Description**

On archives, index, blog/home, and search, the heading above the results should be the only H1.
On single pages and posts, the post title should be the only H1.

When the user adds a pattern that is intended to be one part of the page, that pattern should not contain an H1,
because then by default, there are two H1's, which makes it more difficult for some users to determine what the most important content on the page is.

Landing page patterns, that are intended to be the only pattern on the page. must contain an H1, and only one H1.
(A landing page can be one pattern that is combined of several other patterns)
_Update: These landing pages will need to rely on the user making the right choice, either use the post title, add an H1, or update the levels._

---
Task;:
Review the use of H1's in the current patterns.
Replace H1's with appropriate levels but make sure that the look is the same as before, including: font size, letter spacing, weight etc.

**Comment by carolinan** [comment]
As @richtabor correctly pointed out, even if the landing page patterns were designed to be used on the page template that has no title, the default is to use the template that has the title.

The H1 heading should be replaced on the landing page patterns.
